lib/delayed/performable_method.rb

```diff
@@ -24,13 +24,7 @@ def display_name
     end
 
     def perform
-      return unless object
-
-      if kwargs.nil? || (RUBY_VERSION < '2.7' && kwargs.empty?)
```

Comment:
[I reverted this guard](https://github.com/Betterment/delayed/pull/7/files) after Ruby <3.0 support is not longer required.

Reply:
do we still need the `kwargs.nil?` check?

Reply:
We do, I committed back the support of Ruby 3 positional and keyword arguments.